src/components/Combobox/__stories__/combobox.stories.mdx

```diff
@@ -156,9 +156,14 @@ When having a lot of options, you may use headings to categorize them.
         workspace: { id: "Workspaces", label: "Workspaces" }
       }), []);
       return (
+      <div className="combobox-stories-styles_row">
```

Comment:
please wrap examples by using storyDescription component and add titles: "Categories with divider" and without "Categories without divider".

Reply:
V